sourcecode/apis/contentauthor/database/migrations/2022_03_17_125309_add_is_draft.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class AddIsDraft extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('h5p_contents', function (Blueprint $table) {
```

Comment:
All of these should have `false` as their defaults. The fact that booleans map to `TINYINT(1)` is an implementation detail we should not concern ourselves with, Laravel will do the appropriate conversion.

Reply:
agreed